apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -64,6 +67,12 @@ type VSHNPostgreSQLServiceSpec struct {
 	PostgreSQLSettings runtime.RawExtension `json:"pgSettings,omitempty"`
 }
 
+// VSHNPostgreSQLSchedulingSpec contains settings to control the scheduling of an instance.
+type VSHNPostgreSQLSchedulingSpec struct {
```

Comment:
Isn't the name a bit ambiguous? The customer may think it's PostgreSQL related.

Reply:
Renamed it to `VSHNDBaaSSchedulingSpec`. Does that make more sense?

Reply:
yes, don't forget to update the second PR as well.